tool/tctl/common/auth_command.go

```diff
@@ -276,6 +276,12 @@ func (a *AuthCommand) generateSnowflakeKey(ctx context.Context, clusterAPI auth.
 
 // RotateCertAuthority starts or restarts certificate authority rotation process
 func (a *AuthCommand) RotateCertAuthority(ctx context.Context, client auth.ClientI) error {
+	if a.rotateType == "" {
```

Comment:
Should we just tack `.Required()` on the flag itself rather than allow an invalid configuration to get this far?

Reply:
I originally did that but I wanted to be able to customize the error message to give more context and I didn't see any way to do that with `.Required()`.